crsf.md

```diff
@@ -1121,6 +1122,28 @@ MSP frame over CRSF Payload packing:
 
 ## **0x80 Ardupilot reserved**
 
+## **0x81, 0x82 mLRS reserved**
+
+The 0x80 and 0x81 frames establish the communication of mLRS TX modules with mLRS Configuration Lua scripts running on e.g. EdgeTx and OpenTx remote controllers. They encapsulate mLRS 'mBridge' protocol messages in CRSF frames. The protocol enables the setting of parameters and other functionality, but especially also communicates meta data which are needed for mLRS' parameter model, providing various information to the user, version control, and more features specific to the mLRS link system.
+
+**0x80**: Communication from Lua script/remote controller to TX module. The frame format is:
+
+```mermaid
+flowchart LR
+    0xEE ~~~ id0[Frame Length] ~~~ 0x80 ~~~ 0x4F ~~~ 0x57 ~~~ id1[Command Byte] ~~~ Payload ~~~ CRC
+```
+
+**0x81**: Communication from TX module to Lua script/remote controller. The frame format is:
+
+```mermaid
+flowchart LR
+    0xEA ~~~ id0[Frame Length] ~~~ 0x81 ~~~ id1[Command Byte] ~~~ Payload ~~~ CRC
```

Comment:
This looks like a broadcast frame in its structure, but broadcast frame types should be <0x27 (as per the Broadcast frame section). @kamik2 We may have to restructure this document and extract Project extensions to a new section so that the spec is consistent.